inference-engine/src/offline_transformations/include/moc_transformations.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API MOCTransformations;
```

Comment:
export is not needed since we are working with static library. And `transformations_visibility.hpp` include can be removed as well

Reply:
Done.